### ISSUE TITLE
`clg` semicolon

### ISSUE THREAD

**Issue by mecirmartin** [dsznajder/vscode-react-javascript-snippets#218]
Hey, your extension is great, but i have trouble with `clg` each and every day
When using arrow function and `clg` snippet, the semicolon at the end introduces syntax error

For example:
i am writing code like this: 
```
fetch('x').then(res => clg)
```
this creates
```
fetch('x').then(res => console.log();)
```
which is obviously syntax error and i have to remove the semicolon
is there any chance the semicolon could be removed from the snippet? Thanks for the answer

Would love to create PR for this issue

**Comment by mecirmartin** [comment]
@dsznajder 
Created PR for this issue #219

**Comment by dsznajder** [comment]
@mecirmartin Fixed in v4.3.1
